extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -364,31 +365,34 @@ export class ConsentInstance {
         return;
       }
 
-      const consentStr = consentInfo['consentString'];
-      if (consentStr && consentStr.length > CONSENT_STRING_MAX_LENGTH) {
-        // Verify the length of consentString.
-        // 1024 * 4/3 (base64) = 1336 bytes.
+      const value = composeStoreValue(consentInfo);
+      if (value == null) {
+        // Value can be false, do not use !value check
+        // Nothing to store to localStorage
+        return;
+      }
+
+      // Check size
+      const size = JSON.stringify(
+        dict({
+          [this.storageKey_]: value,
+        })
```

Comment:
I'm curious, why dict, instead of object?

Reply:
It doesn't make much of a difference here given that value is calculated internally. 
But `dict()` returned value makes sure that `JSON.Stringify()` won't error out due to functions or any unparsed stuff in Object. 

I used `dict()` also because that how we store values to localStorage. Kinda want to keep it the same. 
https://github.com/ampproject/amphtml/blob/0a0269ab8079392086fcdd10b87da234163bc6b0/src/service/storage-impl.js#L248-L251

Reply:
Thanks for the explanation!